ethereum/test/test_eth_app.js

```diff
@@ -147,23 +150,24 @@ contract("ETHApp", function (accounts) {
       const recipient = "0xcCb3C82493AC988CEBE552779E7195A3a9DC651f";
 
       // expected amount to unlock
-      const amount = BigNumber(web3.utils.toWei("1", "ether"));
```

Comment:
I think it makes the code easier to maintain and extend if we stick to a single number type, such as `BigNumber`.

I distrust the ad-hoc and unspecified conversions between `String` , `BN`, `BigNumber` and `Number` that occur behind the scenes. It means I never really know the exact type of some variable that holds a number.

Reply:
I agree. I added `BigNumber` back in here in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/47fbb4dd81b9dd8a432ad9e043ec5e6634d897a5.